src/coreclr/nativeaot/Runtime/eventpipe/ds-rt-aot.cpp

```diff
@@ -26,7 +44,54 @@ aot_ipc_get_process_id_disambiguation_key(
     *key = 0;
 
 // Mono implementation, restricted just to Unix
```

Comment:
Thoughts on filing an issue or some other form of tracking all the functionality that is directly copied which would be candidates for sharing?

Reply:
Opened #87069 for this